plugin/secrets_access_token.go

```diff
@@ -141,20 +71,12 @@ func (b *backend) getSecretAccessToken(ctx context.Context, s logical.Storage, r
 		return logical.ErrorResponse(fmt.Sprintf("could not generate token: %v", err)), nil
 	}
 
-	secretD := map[string]interface{}{
-		"token": token.AccessToken,
-	}
-	internalD := map[string]interface{}{
-		"access_token":      token.AccessToken,
-		"key_name":          rs.TokenGen.KeyName,
-		"role_set":          rs.Name,
-		"role_set_bindings": rs.bindingHash(),
-	}
-	resp := b.Secret(SecretTypeAccessToken).Response(secretD, internalD)
-	resp.Secret.TTL = token.Expiry.Sub(time.Now())
-	resp.Secret.Renewable = false
-
-	return resp, err
+	return &logical.Response{
+		Data: map[string]interface{}{
+			"token":      token.AccessToken,
+			"expires_at": token.Expiry,
```

Comment:
What type is this? In my experience, date types tend not to be transported well.

Additionally, if we calculate this as a TTL (time.Now().Unix().Sub(token.Expiry) and return this as a field named `*_ttl`, the Vault CLI will format it nicely (i.e. 30m)

Reply:
I left the expiry but changed it to a unix timestamp, and added token_ttl